spring-jdbc/src/main/java/org/springframework/jdbc/core/namedparam/NamedParameterUtils.java

```diff
@@ -258,8 +258,11 @@ private static int skipCommentsAndQuotes(char[] statement, int position) {
 	 */
 	public static String substituteNamedParameters(ParsedSql parsedSql, @Nullable SqlParameterSource paramSource) {
 		String originalSql = parsedSql.getOriginalSql();
-		StringBuilder actualSql = new StringBuilder();
 		List<String> paramNames = parsedSql.getParameterNames();
+		if(paramNames.isEmpty()) {
```

Comment:
Minor thing: According to our code style, the "if" needs a space before the bracket, i.e. "if (paramNames.isEmpty())". If you could fix this quickly, I'd be happy to merge your pull request unchanged. I'll care for the backport to 4.3.x towards the end of this week then.

Reply:
Done.
Thx for the backport